### DISCUSSION TITLE
Do you use the VCF output option?

### DISCUSSION THREAD

**Discussion by julesjacobsen** [exomiser/Exomiser#544] [Polls]
When the `FULL` analysis option is used Exomiser's memory use for WGS samples can get into the tens of GB. Looking at where all the memory is being used **~50%** is due to storing the output of the HTSJDK (used to model the VCF file) which makes it easier to write the results out as VCF with the Exomiser annotations added.

The easiest option to reduce memory use would be remove the VCF output option entirely. Would this severely impact anyone's workflow?

**Comment by ShrutiMarwaha** [comment]
Yes, we use vcf option to feed prioritized SNV/INDEL to other tools.
